.github/workflows/check-minimal-versions.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow checks that the project builds and tests successfully with the
+# minimal versions of all dependencies that are compatible with the current
+# Cargo.lock file. This is useful for ensuring that the project is not
+# accidentally relying on newer versions of dependencies than it actually
+# needs.
```

Comment:
We already have a standard shared action for checking minimal versions here: https://github.com/RustCrypto/actions/blob/master/.github/workflows/minimal-versions.yml